first_year_scraper/plan.txt

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Should we update this file now as I think few more ToDos have been done, no?

Reply:
Ahh! that wasn't intended to be pushed in the repo. It was just a rough plan to track where I was. Sure! If you want I'll update the file!😅